apps/web/app/page.tsx

```diff
@@ -1,11 +1,9 @@
-import { Button } from "@ui/Button";
-import { Navbar } from "@components/Navbar";
+import NavBar from "@components/Navbar";
 
 export default function Page(): JSX.Element {
   return (
     <main>
-      <Navbar />
-      <Button>Click me</Button>
+      <NavBar />
```

Comment:
ui like Navbar should be kept in `layout.tsx` file instead of `page` as navbar should persist even of the page changes